src/walk.rs

```diff
@@ -349,15 +374,4 @@ pub fn scan(path_vec: &[PathBuf], pattern: Arc<Regex>, config: Arc<FdOptions>) {
             ignore::WalkState::Continue
         })
     });
-
-    // Drop the initial sender. If we don't do this, the receiver will block even
-    // if all threads have finished, since there is still one sender around.
-    drop(tx);
```

Comment:
If I understand this correctly, calling `drop` explicitly is not needed anymore because `tx` is moved into the `spawn_senders` function and dropped at the end of the function scope?

Reply:
That is correct. It's explained in the commit message of a05312e